netlify.toml

```diff
@@ -1,6 +1,6 @@
 [build]
   base    = "docs/"
   publish = "docs/public/"
-  command = "gatsby build --prefix-paths && mkdir -p docs/android && mv public/* docs/android && mv docs public/ && mv public/docs/android/_redirects public"
+  command = "gatsby build --prefix-paths && mkdir -p docs/android && mv public/* docs/android && mv docs public/ && mv public/docs/android/_redirects public && cat README.md | grep -v 'the official docs' > docs/source/essentials/get-started.md'"
```

Comment:
You should invert the commands, `getting-started.md` is an input to `gatsby build` and should be present before the command is executed.